frontend/src/component/Account/Password.jsx

```diff
@@ -95,58 +98,70 @@ export class Password extends React.Component {
         <form className='personaldata__form mr-5'>
           {props.displayAdminInfo === false && (
             <div className='d-flex align-items-center flex-wrap mb-4'>
+              <label>
+                {props.t('Current password:')}
+                <input
+                  className='personaldata__form__txtinput primaryColorBorderLighten form-control'
+                  type='password'
+                  placeholder=''
+                  value={state.oldPassword}
+                  onChange={this.handleChangeOldPassword}
+                  maxLength={513}
+                />
+              </label>
+            </div>
+          )}
+
+          <div className='d-flex align-items-center flex-wrap mb-4'>
+            <label>
+              {props.t('New password:')}
               <input
                 className='personaldata__form__txtinput primaryColorBorderLighten form-control'
                 type='password'
-                placeholder={props.t('Old password')}
-                value={state.oldPassword}
-                onChange={this.handleChangeOldPassword}
+                placeholder=''
```

Comment:
```suggestion
```

Reply:
@raphj thanks i didn't know if i should delete it or not :)